product_docs/docs/pem/9/registering_agent.mdx

```diff
@@ -233,3 +233,31 @@ To use a nonroot user account to register a PEM agent, you must first install th
 
 !!! Note
     Any probes and jobs that require root permission or access to a file owned by another user (for example,  enterprisedb) fail.
+
+## Modifying the pemagent service file to update agent configuration file path
+
+In some situations, you might need to move the agent configuration file from its default location `/usr/edb/pem/agent/etc/agent.cfg` to another location. In that case, you need to update the value of the agent configuration file path in the `pemagent` service file. 
+
+1. Use the following command to modify the `pemagent` service file as a superuser:
+
+    ```ini
+    $ sudo vi  /usr/lib/systemd/system/pemagent.service
```

Comment:
I thought we weren't supposed to include the prompt in code blocks?

Reply:
yes, we should not use prompts in the code blocks. @arup-roy2023, please remove the prompt signs from all the code blocks

Reply:
I have removed the prompt symbol from all the code blocks.